core/server/data/schema.js

```diff
@@ -161,12 +162,23 @@ var db = {
             uuid: {type: 'string', maxlength: 36, nullable: false},
             name: {type: 'string', maxlength: 150, nullable: false, unique: true},
             slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
-            secret: {type: 'string', maxlength: 150, nullable: false, unique: true},
+            secret: {type: 'string', maxlength: 150, nullable: false},
```

Comment:
Yes, every existing installation has the unique constraint. Executing the migration to 004 should remove the constraint from existing databases. This is done here https://github.com/TryGhost/Ghost/blob/master/core/server/data/migration/commands.js#L48, hopefully ;-).